### PR TITLE
Add EIP: Offchain checks for ERC-20 distribution

### PR DESCRIPTION
**Pull request by civia-code**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by civia-code** [comment]
Discussion at https://ethereum-magicians.org/t/eip-7099-off-chain-checks/14526

File: EIPS/eip-7099.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
+---
+eip: 7099
+title: Offchain checks for ERC-20 distribution
```
Comment (lightclient) [inline EIPS/eip-7099.md:3]:
```suggestion
title: Off-chain checks for ERC-20 distribution
```

File: EIPS/eip-7099.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-05-31
+requires: 20, 712
+---
+
+## Abstract
+
+We propose a low cost approach to distribute [ERC-20](./eip-20.md) tokens for high-frequency token issuance use cases such as game rewards, implemented with a one-time deployed Check contract, the off-chain issuing and signing of Checks by the issuers, and batching or bundling of the Checks to mint tokens on-chain by the receivers.
```
Comment (lightclient) [inline EIPS/eip-7099.md:16]: Sentence is a bit of a run on.

File: EIPS/eip-7099.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+    end
+```
+
+## Rationale
+
+The off-chain check workflow, while shifting the main cost of token distribution from the issuers to the receivers, adds flexibilities to token issuance. The issuers don't have to decide beforehand when and how many tokens must be minted to what group of receivers. After they register the tokens, the issuers can write Checks as they go. 
+
+The receivers on the other hand, before minting the tokens on-chain, are assured of the promised tokens with issuers' signatures on the Checks. The receivers accumulate Checks to mint in batch or bundle to save gas, and have the option to mint at a low gas time period.
+
+## Backwards Compatibility
```
Comment (lightclient) [inline EIPS/eip-7099.md:216]:
```suggestion
```
Section is optional, no need to include if nothing relevant is in the section.

**Review by lightclient — CHANGES_REQUESTED** [review]
Seems okay, couple nits. Overall this seems very similar to a payment channel though. Is there a reason to standardize this instead of simply using a state channel? Even if not, what is beneficial about standardizing this?

**Comment by SamWilsn** [comment]
I am closing this pull request because we are in the process of separating EIPs and ERCs into distinct repositories. Unfortunately, as far as we are aware, GitHub does not provide any tools to ease this migration, so every pull request will need to be re-opened manually.

As this is a PR to create / modify an ERC, I will kindly ask you to redirect this to the new repository at [ethereum/ERCs](https://github.com/ethereum/ERCs). **We have prepared [a guide](https://github.com/ethereum/ERCs/issues/8) to help with the process.**

If there is relevant history here, please link to this PR from the new pull request.

On behalf of the EIP Editors, I apologize for this inconvenience.

**Pull request closed without merging**